docs/users/author-guide.md

```diff
@@ -1,28 +1,106 @@
 # Author guide: creating and publishing projects
 
-## Creating projects
+To make it easy for investigators to work with a broad range of model formats, modeling frameworks, simulation types, simulation algorithms, and simulation tools, BioSimulations uses existing community standards for creating and describing projects. BioSimulations uses the Simulation Experiment Description Markup Language (SED-ML) to describe [simulation experiments](#sed-ml) and COMBINE/OMEX archive formats for packaging and distribution of simulation [projects](#combineomex-archives). The OMEX-Meta format is used to describe the [metadata](#metadata) associated with a simulation project. Vega is used to describe charts, plots and [visualizations](#visualizations) of simulation results. More information about BioSimulation's use of these standards, and how to create and publish a project can be found below.
 
-### SED-ML
+## Creating Projects
```

Comment:
Title capitalization should be "Creating projects", consistent with other sections